src/test/rgw/test_rgw_manifest.cc

```diff
@@ -25,8 +25,8 @@
 #define ASSERT_EQ(v, s) if(v != s)cout << "Error at " << __LINE__ << "(" << #v << "!= " << #s << "\n"; \
                                 else cout << "(" << #v << "==" << #s << ") PASSED\n";
 #define EXPECT_EQ(v, s) ASSERT_EQ(v, s)
-#define ASSERT_TRUE(c) if(c)cout << "Error at " << __LINE__ << "(" << #c << ")" << "\n"; \
-                          else cout << "(" << #c << ") PASSED\n";#define EXPECT_TRUE(c) ASSERT_TRUE(c) 
+#define ASSERT_TRUE(c) if(!c)cout << "Error at " << __LINE__ << "(" << #c << ")" << "\n"; \
```

Comment:
@david-z under what circumstances is this chunk compiled? as i found that we have
```c++
#define GTEST
#ifdef GTEST
...
#else
...
```

Reply:
@tchaikov  yes, we used gtest, so this error was never came out. I just noticed this code error by accident, so submit the fix.

Reply:
@david-z thanks. the change per-se looks good apart from some formatting nits..

but, actually, i am in favor of removing them as a whole. these GTEST macro replacements were introduced by #1214.

@yehudasa @jdurgin do you recall why we have them? can we simply ditch these macros?

Reply:
@tchaikov I've no idea where they came from

Reply:
@tchaikov  do we ditch these macros anyway? I think it is totally fine.

Reply:
@david-z agreed, i am all for removing dead code! could you repurpose this PR to do so?

Reply:
Sure, will do.